css/portal-dashboard/progress-legend.less

```diff
@@ -0,0 +1,77 @@
+.legendContainer {
+    font-family: lato, arial, helvetica, sans-serif;
+    font-size: 14px;
+    font-weight: normal;
+    color: #3f3f3f; 
+    border: 1.5px solid #979797;
+    box-sizing: border-box;
+    border-radius: 16px;
+    text-align: center;
+    width: 477px;
+    height: 32px;
+    line-height: 32px;
+    background-color: #ffffff;
+    left: 0px;
+    right:0px;
+    margin-left:auto;
+    margin-right:auto;
+    position: absolute;
+    bottom:-19px;
+  }
+  .legendLabel{
+    display:inline-block;
+    position:relative;
+    bottom:2px;
+  }
+  .legendKey {    
+    display:inline-block;
+    height: 20px;
+    vertical-align: text-bottom;
+    padding-right: 9px;
+  }
+  .legendText {
+    display:inline-block;
+    position:relative;
+    bottom:10px;
+    line-height: 17px;
+    vertical-align: text-bottom;
+    position: relative;
+  }
+  .completed {
+    margin: 5px;
+    position:relative;
+    bottom:3px;
+    box-sizing: border-box;
+    border: 1px solid #ff8415;
```

Comment:
Colors like this can now be pulled out of the new `variables.less` if you rebase onto master.  So this color would be `@cc-orange`.  Just make sure to import the following in the top of the .less file: `@import "./variables";`